### PR TITLE
feat: remove @vercel/og

### PR DESCRIPTION
**Pull request by jxom**

### CODE REVIEW

File: src/utils/elementToSvg.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ...options,
+    width,
+    height,
+    fonts: fonts
+      ? fonts
+      : [
+          {
+            name: "Open Sans",
+            data: await loadGoogleFont({ family: "Open Sans", weight: 600 }),
+            weight: 500,
```
Comment (m4nuC) [inline src/utils/elementToSvg.ts]: This is needed for emojis and multi-language support: https://github.com/vercel/satori/blob/main/playground/pages/index.tsx#L561

File: src/utils/elementToSvg.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export async function elementToSvg(
+  element: HonoElement,
+  options: ElementToSvgOptions = {},
+) {
+  const { fonts, height = 630, width = 1200 } = options;
+
+  const svg = await satori(toReactNode(element), {
+    ...options,
+    width,
+    height,
```
Comment (m4nuC) [inline src/utils/elementToSvg.ts:32]: This is needed for emojis and multi-language support: https://github.com/vercel/satori/blob/main/playground/pages/index.tsx#L561

Comment (jxom) [reply src/utils/elementToSvg.ts:32]: ty

Comment (jxom) [reply src/utils/elementToSvg.ts:36]: done https://github.com/wevm/hono-og/pull/28/commits/c4abb1c05ad8acbec214a572d6ac652973eaba19

**Pull request merged**